specification/security/resource-manager/Microsoft.Security/stable/2020-01-01/automations.json

```diff
@@ -0,0 +1,647 @@
+      "post": {
+        "x-ms-examples": {
+          "Validate the security automation model before create or update": {
+            "$ref": "./examples/Automations/ValidateAutomation_example.json"
+          }
+        },
+        "tags": [
+          "Automations"
+        ],
+        "description": "Validates the security automation model before create or update. Any validation errors are returned to the client.",
+        "operationId": "Automations_Validate",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/AutomationName"
+          },
+          {
+            "$ref": "#/parameters/AutomationInBody"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/AutomationValidationStatus"
+            }
+          },
+          "default": {
+            "description": "Error response that describes why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AutomationList": {
+      "description": "List of security automations response.",
+      "required": [
+        "value"
+      ],
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "The list of security automations under the given scope.",
+          "items": {
+            "$ref": "#/definitions/Automation"
+          }
+        },
+        "nextLink": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The URI to fetch the next page."
+        }
+      }
+    },
+    "Automation": {
+      "type": "object",
+      "description": "The security automation resource.",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "Security automation data",
+          "$ref": "#/definitions/AutomationProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../common/v1/types.json#/definitions/TrackedResource"
+        }
+      ]
+    },
+    "AutomationProperties": {
+      "type": "object",
+      "description": "A set of properties that defines the behavior of the automation configuration. To learn more about the supported security events data models schemas - please visit https://aka.ms/ASCAutomationSchemas.",
+      "properties": {
+        "description": {
+          "type": "string",
+          "description": "The security automation description."
+        },
+        "isEnabled": {
```

Comment:
As a general rule, we prefer string enums to boolean for stability. If you want to add a tertiary state in teh future, changing from bool to string is breaking. You could have a property named `automationState` with values like `[enabled, disabled]`